sources/packages/backend/apps/api/src/route-controllers/announcements/announcements.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get } from "@nestjs/common";
+import BaseController from "../BaseController";
+import { ApiTags } from "@nestjs/swagger";
+import { Announcements } from "@sims/sims-db";
+import { AnnouncementsService } from "../../services";
+import { Public } from "../../auth/decorators/public.decorator";
+
+@Controller("announcements")
+@ApiTags("announcements")
+export class AnnouncementsController extends BaseController {
+  constructor(private readonly announcementsService: AnnouncementsService) {
+    super();
+  }
+
+  @Public()
+  @Get()
+  async getAnnouncements(): Promise<Announcements[]> {
```

Comment:
Should we not return `AnnouncementsAPIOutDTO[]`?

Reply:
Yes and more importantly please ensure that entity models are not directly exposed to APIs. 

Here as per what we follow, create a *APIOutDTO  as a class (not an interface) to define the API.

Reply:
There was a decision to avoid returning arrays in the DTO. The DTO must have an array property.

Reply:
Converting to an array property in a DTO would not be a blocker since we have too many others following the same.